src/ott/solvers/quadratic/gromov_wasserstein.py

```diff
@@ -102,7 +102,7 @@ class GWState(NamedTuple):
       linearization of GW.
     linear_pb: Local linearization of the quadratic GW problem.
     old_transport_mass: Intermediary value of the mass of the transport matrix.
-    keys: Random keys passed to low-rank initializers at every GW iteration
+    rngs: Random keys passed to low-rank initializers at every GW iteration
```

Comment:
why are there multiple keys here ?

Reply:
@michalk8 wdyt?

Reply:
The cleanest way would be to refactor and have only `key: Optional[jax.random.PRNGKeyArray]` that is being split in [iterations](https://github.com/ott-jax/ott/blob/main/src/ott/solvers/quadratic/gromov_wasserstein.py#L373) and passed onto the next state.

Reply:
You mean `rng: Optional[jax.random.PRNGKeyArray]` right? name should be rng?